packages/runtime/src/binding/target-accessors.ts

```diff
@@ -76,17 +75,17 @@ export class DataAttributeAccessor implements DataAttributeAccessor {
   }
 }
 
-export interface StyleAttributeAccessor extends IBindingTargetAccessor<IHTMLElement, 'style', string | IIndexable> {}
+export interface StyleAttributeAccessor extends IBindingTargetAccessor<IHTMLElement, 'style', string | object> {}
 
 @targetObserver()
 export class StyleAttributeAccessor implements StyleAttributeAccessor {
-  public currentValue: string | IIndexable;
-  public defaultValue: string | IIndexable;
+  public currentValue: string | object;
```

Comment:
```suggestion
  public currentValue: string | Record<string, string>;
```
In this unique scenario, we do actually know precisely what type this can (or is allowed to) be.